app/controllers/admin/searchgov_domains_controller.rb

```diff
@@ -21,10 +23,34 @@ class Admin::SearchgovDomainsController < Admin::AdminController
       inline: true,
       confirm: 'Are you sure you want to reindex this entire domain?'
     )
+    config.action_links.add(
+      'confirm_delete',
+      label: 'Delete',
+      type: :member,
+      crud_type: :delete,
+      method: :get,
+      position: :after,
+      inline: false
+    )
     config.update.columns = %i[js_renderer]
     config.columns[:js_renderer].label = 'Render Javascript'
   end
 
+  def confirm_delete
+    render :delete_domain
+  end
+
+  def delete_domain
+    if destroy_domain_confirmation_valid?
+      enqueue_deletion_job
+      flash_message(:success)
```

Comment:
nitpick and I know there will be repetition but in this case I'd prefer setting the flash message explicitly. It'd add less lines of code and we'd not have to ignore the translation check on the i18 task.

```
flash[:succes] = I18n.t(:'.flash_delete_success', domain: @searchgov_domain.domain)
```

Reply:
@stevenbarragan  - I would agree ... Another change from code climate on this to use locales. If that is fine to ignore, I can do.